### ISSUE TITLE
Missing image description to be announced with screen reader

### ISSUE THREAD

**Issue by lizgarciao** [pulibrary/audit-Liz#9]
### URL
https://lizgarciao.github.io/

### Issues
Missing image description to be announced with screen reader

### Severity
High

### WCAG Conformance level (A, AA, AAA)
A

### User impact
User does not know what the image is about because screen reader does not have a description to read.

### Recommended fix
Use alt text to describe image

### Acceptance criteria
- [x] I can successfully use this feature with Voice Over in Safari.
- [x] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave

### Notes
https://www.w3.org/WAI/WCAG21/Techniques/html/H37.html
